demo/index.html

```diff
@@ -79,8 +79,8 @@ <h2 id="iene-heading">initial element, no escape</h2>
   <div id="demo-ifc">
     <h2 id="ifc-heading">initially focused container</h2>
     <p>
-      Initial focus is on the containing element, which has <code>tabindex="-1"</code> and no focus outline;
-      so when you tab through the trap focus does not return to the container.
+      When this focus trap activates, initial focus is on the containing element(which has <code>tabindex="-1"</code> so is not tabbable),
+      so when you tab through the trap, focus does not return to the containing element.
```

Comment:
I feel previous version can be easier to understand by some rewording.

Reply:
What would you think about wording it this way?
```
When this focus trap activates, initial focus is on the containing element (which has <code>tabindex="-1"</code> and is therefore not tabbable).
When you tab through the trap, focus does not return to the containing element.
```
Just a _slight_ tweak. 🙂

Reply:
Looks better than my version